tests/admin_views/tests.py

```diff
@@ -4706,6 +4707,14 @@ def test_readonly_backwards_ref(self):
         response = self.client.get(reverse('admin:admin_views_topping_add'))
         self.assertEqual(response.status_code, 200)
 
+    @override_settings(DEBUG_PROPAGATE_EXCEPTIONS=True)
+    def test_readonly_onetoone_backwards_ref(self):
+        """Regression test for #26060"""
```

Comment:
Trying adding `@override_settings(DEBUG=True)` to the test. The template system might be silencing the exception otherwise and rendering an empty string otherwise.